temperature/models.py

```diff
@@ -0,0 +1,16 @@
+from sqlalchemy import Column, Integer, DateTime, Float, ForeignKey
+from sqlalchemy.orm import relationship
+
+from city.models import City
```

Comment:
Ensure that the `City` model is correctly imported from the `city.models` module. This is crucial for establishing the relationship between the `Temperature` and `City` models.